transport-native-epoll/src/main/c/io_netty_channel_unix_Socket.c

```diff
@@ -38,6 +38,8 @@ static jmethodID netUtilClassIpv4PreferredMethodId = NULL;
 static int socketType;
 static char* nettyClassName = NULL;
 static const char* ip4prefix = "::ffff:";
+static const char wildcardAddress[] = { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00 };
```

Comment:
`char` -> `unsigned char` according to http://man7.org/linux/man-pages/man7/ipv6.7.html?